corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -269,17 +269,19 @@ class ParentCasePropertyBuilder(object):
     Full functionality is documented in the individual methods.
 
     """
-    def __init__(self, app, defaults=(), per_type_defaults=None, include_parent_properties=True,
-            exclude_invalid_properties=False):
+    def __init__(self, app, defaults=(), per_type_defaults=None,
+                 include_parent_properties=True, exclude_invalid_properties=False,
+                 traverse_apps=True):
```

Comment:
Would be good to document why `traverse_apps` exists.

Reply:
:+1: I'm hoping my next PR will remove it

Reply:
even better!